lib/ProductOpener/Import.pm

```diff
@@ -1921,14 +1935,7 @@ sub import_csv_file($) {
 
 									$log->debug("download image file", { file => $file, image_url => $image_url }) if $log->is_debug();
 
-									require LWP::UserAgent;
-
-									my $ua = LWP::UserAgent->new(timeout => 10);
-									
-									# Some platforms such as CloudFlare block the default LWP user agent.
-									$ua->agent(lang('site_name') . " (https://$server_domain)");
-
-									my $response = $ua->get($image_url);
+									my $response = download_image($image_url);
```

Comment:
Good idea to make a function, it makes things more readable, in addition to being needed for the mock.